src/assets/icons/location.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+    <path stroke="#d9d9d9" fill="#d9d9d9" d="M12 2C8.13 2 5 5.13 5 9c0 5.25 7 13 7 13s7-7.75 7-13c0-3.87-3.13-7-7-7zm0 9.5c-1.38 0-2.5-1.12-2.5-2.5s1.12-2.5 2.5-2.5 2.5 1.12 2.5 2.5-1.12 2.5-2.5 2.5z"/>
+    <path d="M0 0h24v24H0z" fill="none"/>
+</svg>
```

Comment:
[location-with-shadow.svg.zip](https://github.com/Canary-PrintDesign/canswe/files/2003489/location-with-shadow.svg.zip)

add the attached in as well.